### PR TITLE
Updated the Session class.

### PR DESCRIPTION
**Pull request by pasang2044**

### CODE REVIEW

File: src/io/keyword/easyevents/Sessions.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+/**
+ * Class Description
+ * <p>
+ * Created By: Justin Reeves, Pasang Sherpa, Xiaotian Wang (Xander)
+ * Created On: 3/26/2021
+ */
+class Sessions {
+
+    // FIELDS
+    private LocalTime initialTime;
```
Comment (justin-reeves) [inline src/io/keyword/easyevents/Sessions.java:14]: Should we have initialTime as part of EventLog?

File: src/io/keyword/easyevents/Sessions.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+import java.time.LocalTime;
+
+/**
+ * Class Description
+ * <p>
+ * Created By: Justin Reeves, Pasang Sherpa, Xiaotian Wang (Xander)
+ * Created On: 3/26/2021
+ */
+class Sessions {
```
Comment (justin-reeves) [inline src/io/keyword/easyevents/Sessions.java:11]: This needs to be public

File: src/io/keyword/easyevents/Sessions.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    private String sessionName;
+    private static final Sessions instance = new Sessions();
+
+
+    // CONSTRUCTORS
+
+
+    private Sessions() {
+    }
+
```
Comment (justin-reeves) [inline src/io/keyword/easyevents/Sessions.java:25]: Need ctor for (String name), (LocalTime initialTime), (String name, LocalTime intialTime)

File: test/io/keyword/easyevents/SessionsTest.java [hunk 4]
```diff
@@ -4,9 +4,9 @@
 
 import static org.junit.Assert.*;
 
-public class VideoRecordingTest {
+public class SessionsTest {
 
     @Before
-    public void setUp() {
+    public void setUp() throws Exception {
```
Comment (justin-reeves) [inline test/io/keyword/easyevents/SessionsTest.java:10]: Can probably get rid of 'throws'

**Review by justin-reeves — APPROVED** [review]

**Review by wang-xiaotian — APPROVED** [review]

**Pull request merged**